web/controllers/rules/list_rules.ex

```diff
@@ -14,7 +14,8 @@ defmodule Dwylbot.Rules.List do
         ]
       "pull_request" ->
         [
-          Dwylbot.Rules.PR.NoDescription
+          Dwylbot.Rules.PR.NoDescription,
+          Dwylbot.Rules.PR.MergeConflict
```

Comment:
I might have these aliased at the top of the file as recommended here in the [credo-styleguide](https://github.com/rrrene/elixir-style-guide#credos-elixir-style-guide): ["When developing applications, try to alias all used modules"](https://github.com/rrrene/elixir-style-guide#software-design)